.github/workflows/docker_ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           RMQHOST: rmq
           INTEGRATION_TEST_SETUP: true
           INTEGRATION_TEST_SEED: '/code/spec/data/integration/seed_for_unified_wh.rb'
-          RAILS_ENV: test
```

Comment:
And this as test?

Reply:
This is the issue with views_schema in unified warehouse. 

Views schema generates a dump of the view definition from the local database, and this definition is what we use for local development and for testing. The problem is that the view definition refers to a specific database (either development or test) when we are joining with event warehouse; I choose the option of generating the view from development as the main reason we want the definition is for development. The problem we have is that then the view is created using development databases for event warehouse, so then we need to create the development database for event warehouse.